### ISSUE TITLE
bug: aria-hidden=true should be removed

### ISSUE THREAD

**Issue by titanism** [Flet/markdown-it-github-headings#20]
This line needs to be removed: <https://github.com/Flet/markdown-it-github-headings/blob/d4c008d06ee692a6a564858bd78e63bd58e8ace0/index.js#L62>

Note that you already have `aria-hidden=true` on the SVG, which should suffice.

Google's PageSpeed Insights deems this current behavior negatively.

<img width="1006" alt="Screen Shot 2023-01-08 at 8 17 14 PM" src="https://user-images.githubusercontent.com/101466223/211231740-d9046680-ee58-4e2a-b81f-a27f7bbee02e.png">

**Comment by titanism** [comment]
Someone should probably also notify GitHub support of this since this is mirrored from them.
